extensions/wikia/Discussions/maintenance/dumpForumData.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Dumps the Forum data for the site selected by setting the SERVER_ID variable
+ *
+ * @ingroup Maintenance
+ */
+
+error_reporting(E_ALL);
+
+require_once( __DIR__ . '/../../../../maintenance/Maintenance.php' );
+include_once( __DIR__ . '/ForumDumper.php' );
+
+class DumpForumData extends Maintenance {
+	/** @var  \Discussions\ForumDumper */
+	private $dumper;
+	private $fh;
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = 'Dumps a set of INSERT statements suitable for importing into Discussion "import" tables';
+		$this->addArg( 'out', "Output file for SQL statements", $required = false );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$required" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)